test/studies/isx/isx-no-return.graph

```diff
@@ -0,0 +1,5 @@
+perfkeys: total =, bucket count =, bucket offset =, bucketize =, exchange =, count keys =
+files: isx-no-return.dat
```

Comment:
Really?  But this is a copy of my other two graph files which are working without that...?